creator-node/src/components/tracks/tracksComponentService.js

```diff
@@ -61,6 +57,44 @@ const handleTrackContentRoute = async ({ logContext }, requestProps) => {
   return resp
 }
 
+async function handleTranscodeAndSegment(
```

Comment:
this wrapper layer is confusing to me - are we wrapping this transparently to follow convention w other fns in this componentservice file? it seems unnecessary otherwise but i could be missing context.